content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,363 @@
+
+    In Kubernetes environments, execute the following command to determine the Kiali URL:
+
+    {{< text bash >}}
+    $ KIALI_URL="http://$(kubectl -n istio-system get svc istio-ingressgateway -o jsonpath='{.spec.clusterIP}'):15029"
+    $ echo $KIALI_URL
+    http://172.30.141.9:15029
+    {{< /text >}}
+
+    Visit `$KIALI_URL` in your web browser, and in the login screen enter the
+    credentials that you stored in the Kiali secret you created above. If you
+    used the example secret above, you should enter a username of `admin` with
+    a password of `mysecret`.
+
+1.  Get an Overview of Your Mesh
+
+    Once you log in, you will initially be brought to the Overview page showing
+    you all the namespaces which have services in your mesh. The page will look
+    similar to:
+
+    {{< image width="75%" ratio="58%"
+    link="./kiali-overview.png"
+    caption="Example Overview"
+    >}}
+
+1.  View A Namespace Graph
+
+    Click on a namespace to see a graph of all services in that namespace.  For
+    example, if you have the Bookinfo sample application installed, click on
+    `bookinfo` in the Bookinfo namespace card.  You will see a graph similar
+    to:
+
+    {{< image width="75%" ratio="89%"
+    link="./kiali-graph.png"
+    caption="Example Graph"
+    >}}
+
+1.  View Summary Metrics
+
+    Select any node or edge in the graph and notice the summary details panel
+    on the right will display additional data about that selected node or edge.
+
+1.  Experiment with different graph types.
+
+    You can visualize the graph using several different types of graphs such
+    as the "App", "Versioned App", and "Workload" graph types.
+
+    1. Select the "App" graph type from the Graph Type drop down menu. This
+    aggregates all versions of an app into a single graph node. For example,
+    the single "reviews" node represents the three versions of the reviews
+    app).
+    {{< image width="75%" ratio="35%"
+    link="./kiali-app.png"
+    caption="Example App Graph"
+    >}}
+
+    1. Select the "Versioned App" graph type from the Graph Type drop down
+    menu. This shows a node for each version of an app, but all versions of a
+    particular app are grouped together. For example, the "reviews" group box
+    that contains the three nodes represents the three versions of the
+    "reviews" app).
+    {{< image width="75%" ratio="67%"
+    link="./kiali-versionedapp.png"
+    caption="Example Versioned App Graph"
+    >}}
+
+    1. Select the "Workload" graph type from the Graph Type drop down menu.
+    This shows a node for each workload in your service mesh. This graph type
+    does not require you to use the "app" and "version" labels so if you opt to
+    not use those labels on your components, this is the graph type you will
+    use.
+    {{< image width="70%" ratio="76%"
+    link="./kiali-workload.png"
+    caption="Example Workload Graph"
+    >}}
+
+1. Examine Details About The Istio Configuration.
```

Comment:
This step could probably be broken into several separate steps:
View Applications
View Application details
View Workloads
View Workload details
View Services
View Service Details

Reply:
I was in a quandary - not sure how much detail to get into. This istio.io Task is really just supposed to show the user "this is how you can visualize your mesh" but not "these are all the things Kiali can do." So I wasn't sure how much detail I should get into and how many of the different screens I was going to introduce or tell the user to go to. I think going to all of those in separate steps would be overwhelming to a user who just wants an introduction to Kiali and just wants to know "what does my mesh look like?"

Reply:
Fair enough.  I wasn't sure how much detail you wanted to go into either.